### ISSUE TITLE
md-contact-chips search list displaying blank on the demo itself

### ISSUE THREAD

**Issue by KSanthanam** [angular/material#4660]
I was getting blank search list in my md-contact-chips. Then I tested on the demo of the official site itself. which, the same problem exists on the site example https://material.angularjs.org/latest/#/demo/material.components.chips 

<img width="809" alt="screen shot 2015-09-16 at 2 56 01 pm" src="https://cloud.githubusercontent.com/assets/11584231/9896704/71511d4a-5c83-11e5-980c-d740adecc552.png">

**Comment by alexitooi** [comment]
+1, same with autocomplete

**Comment by topherfangio** [comment]
This should be fixed in master already due to #4391:

https://material.angularjs.org/HEAD/#/demo/material.components.chips

Note the difference between `HEAD` [master] and `latest` [0.11.0/the latest official release] in the URL.

Are you still seeing it on the demo site? If so, what browser are you using? I can't reproduce it in Chrome.

**Comment by KSanthanam** [comment]
You are right. /latest/..... demo site has the bug while /HEAD/... demo site doesn't. I am using Chrome Version 45.0.2454.85 (64-bit) on Mac. 

How do I access the HEAD code .Currently I use the link
https://ajax.googleapis.com/ajax/libs/angular_material/0.11.0/angular-material.min.js

I don't want to do bower. Is it possible to access through URL?

**Comment by topherfangio** [comment]
Yup, you can access it via: https://material.angularjs.org/HEAD/angular-material.css and https://material.angularjs.org/HEAD/angular-material.js.

The easiest way to get at this is to open the HEAD version of the site, click a demo, click the "Edit on Codepen" link in the top right and then click the "Settings" button when that opens. It should show you all the URLs you need for that version.

This changes with every single build, though, so you definitely don't want to use it in production; it's just for testing and Codepens generally.

**Comment by KSanthanam** [comment]
@topherfangio Thank you for that. I have the icons in the autocomplete list is mis-aligned. What is the CSS I should adjust. Because it pops as autocomplete I am not able to access the stylesheet when it comes visible. 
<img width="307" alt="screen shot 2015-09-17 at 12 15 25 am" src="https://cloud.githubusercontent.com/assets/11584231/9907238/80bd3338-5cd1-11e5-8d8d-ce1aaa5a151d.png">

**Comment by KSanthanam** [comment]
@topherfangio I found out. it is .md-contact-suggestion

**Comment by alexitooi** [comment]
@topherfangio my time is very limited to provide a codepen which reproduces the issue. But I am using an autocomplete with ng-repeat, something like this:

```javascript
<table>
<tr ng-repeat="customer in searchModel.customers">
<td>
          <md-autocomplete required
                           md-selected-item="customer.vb"
                           ng-disabled="!customer.selected"
                           md-search-text="customer.searchText"
                           md-items="vb in searchModel.vbs"
                           md-item-text="vb.searchName"
                           md-selected-item-change="selectVb(customer, vb)"
                           md-search-text-change="searchVbs(customer)">
            <md-item-template>
              <span md-highlight-text="customer.searchText">{{vb.searchName}}</span>
            </md-item-template>
            <md-not-found>{{'dialog.addcustomer.novbfound' | translate}}</md-not-found>
          </md-autocomplete>
        </td>
</tr>
</table>
```

With that code on master branch I will not see the text of {{vb.searchName}}, but the items can be selected. Same happens on my chip code. With the release branch 0.11.0 everything works as expected?!?

**Comment by topherfangio** [comment]
@alexitooi Indeed; this seems like exactly #4667.

**Comment by topherfangio** [comment]
@alexitooi A fix for #4667 was just pushed to master (8849213ce42fa98169b6fbbb0805feb95a6ba678). Can you see if the new version fixes your issue and allows your `{{vb.searchName}}` again?

**Comment by alexitooi** [comment]
great thank you, now autocomplete and chip work again (highlight as well=)
